cargo/spec/dependabot/cargo/file_parser_spec.rb

```diff
@@ -405,12 +405,6 @@
                     file: "Cargo.toml",
                     groups: ["workspace.dependencies"],
                     source: nil
-                  },
-                  {
-                    requirement: nil,
-                    file: "lib/inherit_ws_dep/Cargo.toml",
```

Comment:
`cargo/spec/fixtures/manifests/workspace_dependencies_child` has `log.workspace = true` (old style that's supported), but this test asserts that both the child and the parent, `cargo/spec/fixtures/manifests/workspace_dependencies_root` where the dependency version is actually defined (`log = "=0.4.0"`) should be the only requirement.